README.rst

```diff
@@ -1,1134 +1,127 @@
-
-then you are probably using the default version of Python that came with
-your Mac. This is outdated. So make sure you install Python again using
-homebrew and try again:
-
-.. code:: bash
-
-   $ brew install python
-
-Once installed then repeat the `Installation process <#windows-linux-macos-with-pip>`_
-
-Project Status
---------------
-
--  [ ] Python Versions support
-
-   -  [x] Python 2.7
-   -  [x] Python 3.6
-
--  [ ] Supported AWS Lambda Runtimes
-
-   -  [x] ``nodejs``
-   -  [x] ``nodejs4.3``
-   -  [x] ``nodejs6.10``
-   -  [x] ``nodejs8.10``
-   -  [x] ``java8``
-   -  [x] ``python2.7``
-   -  [x] ``python3.6``
-   -  [ ] ``dotnetcore1.0``
-   -  [x] ``dotnetcore2.0``
-   -  [x] ``dotnetcore2.1``
-
--  [x] AWS credential support
--  [x] Debugging support
--  [x] Inline Swagger support within SAM templates
--  [x] Validating SAM templates locally
--  [x] Generating boilerplate templates
-
-   -  [x] ``nodejs``
-   -  [x] ``nodejs4.3``
-   -  [x] ``nodejs6.10``
-   -  [x] ``nodejs8.10``
-   -  [x] ``java8``
-   -  [x] ``python2.7``
-   -  [x] ``python3.6``
-   -  [x] ``dotnetcore1.0``
-   -  [x] ``dotnetcore2.0``
-
-Contributing
-------------
-
-Contributions and feedback are welcome! Proposals and pull requests will
-be considered and responded to. For more information, see the
-`CONTRIBUTING <CONTRIBUTING.md>`__ file.
-
-A special thank you
--------------------
-
-SAM CLI uses the open source
-`docker-lambda <https://github.com/lambci/docker-lambda>`__ Docker
-images created by `@mhart <https://github.com/mhart>`__.
-
-
-.. raw:: html
-
-   <!-- Links -->
-
-.. |Build Status| image:: https://travis-ci.org/awslabs/aws-sam-cli.svg?branch=develop
-.. |Apache-2.0| image:: https://img.shields.io/npm/l/aws-sam-local.svg?maxAge=2592000
-.. |Contributers| image:: https://img.shields.io/github/contributors/awslabs/aws-sam-cli.svg?maxAge=2592000
-.. |GitHub-release| image:: https://img.shields.io/github/release/awslabs/aws-sam-cli.svg?maxAge=2592000
-.. |PyPI version| image:: https://badge.fury.io/py/aws-sam-cli.svg
-
+.. raw:: html
+
+   <p align="center">
+
+.. raw:: html
+
+   </p>
+
+==============
+SAM CLI (Beta)
+==============
+
+|Build Status| |Apache-2.0| |Contributers| |GitHub-release| |PyPI version|
+
+`Join the SAM developers channel (#samdev) on
+Slack <https://awssamopensource.splashthat.com/>`__ to collaborate with
+fellow community members and the AWS SAM team.
+
+``sam`` is the AWS CLI tool for managing Serverless applications
+written with `AWS Serverless Application Model
+(SAM) <https://github.com/awslabs/serverless-application-model>`__. SAM
+CLI can be used to test functions locally, start a local API Gateway
+from a SAM template, validate a SAM template, fetch logs, generate sample payloads
+for various event sources, and generate a SAM project in your favorite
+Lambda Runtime.
+
+-  `SAM CLI (Beta) <#sam-cli-beta>`__
+-  `Main features <#main-features>`__

```

Comment:
@jschwarzwalder, to reduce the need to navigate to each of these docs to learn what they are about, can you please add a couple of lines describing the content of each of these files?

Reply:
I simply removed the sections from this Readme and put them in separate files. 

Previously there was simply a list of headings. 

Do you want information for each of these headings? Better names for the pages? 

Should I remove the links from these three and instead link to the Getting Started Section where I can add more descriptive titles?